ci/test_wheel.sh

```diff
@@ -4,9 +4,10 @@
 set -eou pipefail
 
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
-RAPIDS_PY_WHEEL_NAME="rmm_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
+WHEELHOUSE="${PWD}/dist/"
+RAPIDS_PY_WHEEL_NAME="${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 python "${WHEELHOUSE}"
```

Comment:
I would assume that for most libraries, the C++ wheel needs to be downloaded as well, but we get away without it here since librmm is header-only. Is that correct? Maybe worth pointing this out to others working on C++ wheels.

Reply:
Yes, that's right. librmm is only a build-time dependency of rmm, not a runtime dependency. I can make sure that's clear to others too.